d1985r0.md

```diff
@@ -414,6 +423,163 @@ reflection.
 TODO. Concievably simplifies the implementation and makes it far more
 general.
 
+# Covariance and Contravariance
+
+Consider the two concepts:
+
+```cpp
+template <typename T> concept A = true;
```

Comment:
Do you mean that the name of this concept should be is_a. If not I don't see any connection to the covariance part of the code.

Reply:
I renamed is_a to A and is_b to B at some point and didn't fix this. nice spot!